tests/pkg/expectations-builders/formation_assignment.go

```diff
@@ -0,0 +1,109 @@
+package expectations_builders
+
+import (
+	"github.com/kyma-incubator/compass/tests/pkg/fixtures"
+	"k8s.io/utils/strings/slices"
+)
+
+type FAExpectationsBuilder struct {
+	expectations map[string]map[string]fixtures.AssignmentState
+}
+
+func NewFAExpectationsBuilder() *FAExpectationsBuilder {
+	return &FAExpectationsBuilder{
+		expectations: map[string]map[string]fixtures.AssignmentState{},
+	}
+}
+
+func (b *FAExpectationsBuilder) GetExpectations() map[string]map[string]fixtures.AssignmentState {
+	return b.expectations
+}
+
+func (b *FAExpectationsBuilder) GetExpectedAssignmentsCount() int {
+	count := 0
+	for _, val := range b.expectations {
+		count += len(val)
+	}
+	return count
+}
+
+func (b *FAExpectationsBuilder) getCurrentParticipants() []string {
+	participants := make([]string, 0, len(b.expectations))
```

Comment:
Could we rename that to `participantIDs` or something similar? Also, the nem of the function as well - `getCurrentPariticipantIDs`?